src/math/Kconfig

```diff
@@ -22,15 +22,35 @@ config POWER_FIXED
 
 
 config BINARY_LOGARITHM_FIXED
-       bool "Binary Logarithm function"
-       default n
-       help
+	bool "Binary Logarithm function"
+	default n
+	help
 	  This option builds Binary Logarithm function (log2n) is the logarithm to
 	  the base 2. The binary logarithm of x is the power to which the number 2
 	  must be raised to obtain the value x. Base 2 logarithm function is implemented
 	  with a short lookup table. (log2n) operates for a range of 32 bit width size
 	  i.e. 1 to 4294967295.
 
+config NATURAL_LOGARITHM_FIXED
+	bool "Natural Logarithm function"
+	default n
+	help
+	  This option builds Natural Logarithm function (loge(N)) is the logarithm to
+	  the base e. The natural logarithm of x is the power to which the number e
+	  must be raised to obtain the value x. Base e logarithm function is implemented
```

Comment:
According to Google there are 11 million pages explaining what the Natural Logarithm is so I don't think the maths reminder is needed. Also, someone configuring `src/math/Kconfig` should have a minimum education in maths. It's been a long time but I think this was in High School?